ssl/t1_lib.c

```diff
@@ -3238,7 +3267,10 @@ static const tls12_hash_info tls12_md_info[] = {
     {NID_sha224, 112, EVP_sha224},
     {NID_sha256, 128, EVP_sha256},
     {NID_sha384, 192, EVP_sha384},
-    {NID_sha512, 256, EVP_sha512}
+    {NID_sha512, 256, EVP_sha512},
+    {NID_id_GostR3411_94,       128, md_gost94},
+    {NID_id_GostR3411_2012_256, 128, md_gost2012_256},
+    {NID_id_GostR3411_2012_512, 256, md_gost2012_512},
```

Comment:
I can organize a lookup table here or just make an array , but i still do not understand how to deal with the previous place near the line 957.